go.sum

```diff
@@ -1,5 +1,7 @@
 cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
 github.com/BurntSushi/toml v0.3.1/go.mod h1:xHWCNGjB5oqiDr8zfno3MHue2Ht5sIBksp03qcyfWMU=
+github.com/capgemini-engineering/go-diameter v1.0.0 h1:CMptV+HT5e6FPn9fjgHS42u/c74GGNnGm2pQjkLEals=
```

Comment:
this might be due to your local dev setup, can you please clean it up?

Reply:
Change have incorporated

Reply:
@lxfontes please see and let me know if there are any other comments